superset/assets/src/modules/colors.js

```diff
@@ -593,3 +593,12 @@ export function hexToRGB(hex, alpha = 255) {
   const b = parseInt(hex.slice(5, 7), 16);
   return [r, g, b, alpha];
 }
+
+
+export const rgbToHex = function (color) {
+    console.log(color);
```

Comment:
oops

Reply:
Can use `d3-color` instead of writing a new module